src/Components/Aspire.Azure.Messaging.EventHubs/AssemblyInfo.cs

```diff
@@ -20,6 +20,9 @@
 [assembly: ConfigurationSchema("Aspire:Azure:Messaging:EventHubs:PartitionReceiver", typeof(AzureMessagingEventHubsPartitionReceiverSettings))]
 [assembly: ConfigurationSchema("Aspire:Azure:Messaging:EventHubs:PartitionReceiver:ClientOptions", typeof(PartitionReceiverOptions))]
 
+[assembly: ConfigurationSchema("Aspire:Azure:Messaging:EventHubs:EventHubBufferedProducerClient", typeof(AzureMessagingEventHubsBufferedProducerSettings))]
```

Comment:
@eerhardt I don't think this was ignored on purpose, correct?

Reply:
The support for this client was added by someone other than myself, with a small PR. They weren't familiar with the codebase and I would say it's 99% sure they didn't know about this plumbing. I can't remember who it was, but it would probably show up in PRs if you search for that client type.